applications/dashboard/views/modules/guest.php

```diff
@@ -1,24 +1,27 @@
 <?php if (!defined('APPLICATION')) exit(); ?>
 <div class="Box GuestBox">
-    <h4><?php echo t('Howdy, Stranger!'); ?></h4>
+    <h4 class="GuestBox-title">
+        <?php echo t('Howdy, Stranger!'); ?>
+    </h4>
+    <p class="GuestBox-message">
+        <?php echo t($this->MessageCode, $this->MessageDefault); ?>
+    </p>
 
-    <p><?php echo t($this->MessageCode, $this->MessageDefault); ?></p>
-
-    <p><?php $this->fireEvent('BeforeSignInButton'); ?></p>
+    <p class="GuestBox-beforeSignInButton">
+        <?php $this->fireEvent('BeforeSignInButton'); ?>
+    </p>
 
     <?php
-    $signInUrl = signInUrl($this->_Sender->SelfUrl);
-
-    if ($signInUrl) {
-        echo '<div class="P">';
+        if ($this->data('signInUrl')) {
```

Comment:
Can you put this at the same indentation level as before so the diff is cleaner?